### ISSUE TITLE
Non-text info files are shown as text

### ISSUE THREAD

**Issue by psonice** [demozoo/demozoo#385]
If an info file isn't basic text (in my case it's .pdf, but I suspect this'll apply to html, rtf etc.) it uploads fine but when viewed the raw text is displayed.

This would be better handled by just linking to the file if it's not .txt, .diz etc. so the browser can handle it. 

Example: https://demozoo.org/graphics/205206/ (see readme.pdf)

**Comment by glennlunder** [comment]
This feature is specifically intended for raw text, so pdf's are really outside its scope unfortunately. If there is confusion about what "info files" entails, perhaps we need some better signposting for that.

I took the liberty of deleting the file from the site, since it won't display properly as you duly noted.

**Comment by nswaldman** [comment]
I feel that we should consider modern file formats. Info being plain text is a tradition, but not a standard. The solution presented by psonice (i.e. based on mime/type, let the browser present the file) is worth looking into.

**Comment by psonice** [comment]
Yes - plain text is tradition, but it limits us to ascii art, we can't include modern design and artwork, we can't include e.g. a "making of" section. An info file that's limited to text is limited in the kind of information we can provide :) (And we can't even specify the font and text width for ascii art!)
